<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fs_screen</class>
 <widget class="QMainWindow" name="fs_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Food and Storage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="toggle_lang_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle English/German</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_food_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Food</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_food_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Food</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_food_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Food</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="food_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>611</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Supply (kg)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supplier</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supplier Phone Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supplier Email</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
